codespell_lib/data/dictionary.txt

```diff
@@ -25756,8 +25799,12 @@ segmetned->segmented
 segmetns->segments
 segument->segment
 seguoys->segues
+segway->segue
```

Comment:
I'm not so sure about this one:
https://www.merriam-webster.com/words-at-play/segue-segway-commonly-confused
https://en.wikipedia.org/wiki/Segway